src/main/java/org/mitre/synthea/export/CSVExporter.java

```diff
@@ -383,7 +385,15 @@ public void export(Person person, long time) throws IOException {
       String payerID = encounter.claim.payer.uuid;
 
       for (HealthRecord.Entry condition : encounter.conditions) {
-        condition(personID, encounterID, condition);
+    	  /*condition to ignore codes other then retrieved from terminology url*/
+    	  if (!StringUtils.isEmpty(Config.get("generate.terminology_service_url"))) {
+    		  if (RandomCodeGenerator.selectedCodes.stream()
```

Comment:
Looks like there's a mix of spaces and tabs here. Please follow whatever whitespace formatting that the file already had.

Reply:
@hankwallace I will update whitespace formatting in my eclipse so will not have this sort of issues again.

Reply:
@hankwallace, this white-space issue fixed on my eclipse, so now there won't be any issue around this.